From 1680a2407d930d4c101b6d6d1828139597e34e12 Mon Sep 17 00:00:00 2001
From: Max Wittal <max.wittal@mwittal.de>
Date: Thu, 2 May 2024 15:30:00 +0700
Subject: [PATCH] added missing <cstdint> include, required by GCC 13

---
 include/libbech32/bech32.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/libbech32/bech32.h b/include/libbech32/bech32.h
index bc1752c..758d7e8 100644
--- a/include/libbech32/bech32.h
+++ b/include/libbech32/bech32.h
@@ -5,6 +5,7 @@
 
 #include <string>
 #include <vector>
+#include <cstdint>
 
 
 namespace bech32 {
@@ -75,6 +76,7 @@ namespace bech32 {
 
 #ifndef __cplusplus
 #include <stddef.h>
+#include <stdint.h>
 #endif
 
 #ifdef __cplusplus
